gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.            connector.close()
</pre><p><code>connector</code> isn't used elsewhere in this function. Is this intentional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Passed it in the function args, thanks for spotting</p>